usr/lib/api/statistics.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2021
+ *
+ * This program is provided under the terms of the Common Public License,
+ * version 1.0 (CPL-1.0). Any use, reproduction or distribution for this
+ * software constitutes recipient's acceptance of CPL-1.0 terms which can be
+ * found in the file LICENSE file or at
+ * https://opensource.org/licenses/cpl1.0.php
+ */
+#ifndef OCK_STATISTICS_H
+#define OCK_STATISTICS_H
+
+#include <pkcs11types.h>
+#include "slotmgr.h"
+#include "mechtable.h"
+#include "supportedstrengths.h"
+
+/*
+ * Statistics are collected in a shared memory segment per user.
+ * The statistics shared memory segment has the following layout:
+ * - For each configured slot:
+ *    - For each mechanisms supported:
+ *       - one counter (counter_t) for non-key mechanisms (strength=0)
+ *       - for each supported strength one counter (counter_t each)
+ *
```

Comment:
one counter for each supported strength (?)

Reply:
fixed